src/main/java/Deadline.java

```diff
@@ -0,0 +1,13 @@
+public class Deadline extends Task {
+    String dueDate;
```

Comment:
Perhaps you want to make this field private? By default it is default access modifier :D